js/constants/config.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     authUrl: (userId) => `${vaultHost}/v1/users/${userId}`,
     replacementUrl: adHost
   },
-  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd'
+  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd',
+  PDFJSExtensionId: 'ckglimbbooghmggmkikhbolcfmgniemo'
```

Comment:
in chrome, the PDF viewer origin is `chrome-extension://oemmndcbldboiebfnladdacbdfmadadm` which makes more sense